src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -171,6 +171,52 @@
   <data name="AbstractTypesShouldNotHaveConstructorsMessage" xml:space="preserve">
     <value>Abstract type '{0}' should not have public constructors</value>
   </data>
+  <data name="LoggerMessageDiagnosticNumericsInFormatStringTitle" xml:space="preserve">
+    <value>Numerics should not be used in logging format string</value>
+  </data>
+  <data name="LoggerMessageDiagnosticNumericsInFormatStringMessage" xml:space="preserve">
+    <value>Numerics should not be used in logging format string</value>
+  </data>
+  <data name="LoggerMessageDiagnosticNumericsInFormatStringDescription" xml:space="preserve">
+    <value>Numerics should not be used in logging format string</value>
+  </data>
+  <data name="LoggerMessageDiagnosticFormatParameterCountMismatchTitle" xml:space="preserve">
+    <value>Logging format string parameter count mismatch</value>
+  </data>
+  <data name="LoggerMessageDiagnosticFormatParameterCountMismatchMessage" xml:space="preserve">
+    <value>Logging format string parameter count mismatch</value>
+  </data>
+  <data name="LoggerMessageDiagnosticFormatParameterCountMismatchDescription" xml:space="preserve">
+    <value>Logging format string parameter count mismatch</value>
+  </data>
+  <data name="LoggerMessageDiagnosticUseCompiledLogMessagesTitle" xml:space="preserve">
+    <value>Use compiled log messages</value>
+  </data>
+  <data name="LoggerMessageDiagnosticUseCompiledLogMessagesMessage" xml:space="preserve">
+    <value>For improved performance, use pre-compiled log messages instead of calling '{0}' with a string message.</value>
+  </data>
+  <data name="LoggerMessageDiagnosticUseCompiledLogMessagesDescription" xml:space="preserve">
+    <value>For improved performance, use pre-compiled log messages instead of calling '{0}' with a string message.</value>
+  </data>
```

Comment:
There shouldn't be a placeholder here.

Reply:
I found some existing cases where placeholders are used in the resx in this repo.

```
> git grep -c "{0}" -- *.resx

src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx:39
src/Microsoft.CodeAnalysis.BannedApiAnalyzers/Core/BannedApiAnalyzerResources.resx:3
src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx:134
src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx:149
src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/MicrosoftNetFrameworkAnalyzersResources.resx:25
src/PerformanceSensitiveAnalyzers/CSharp/Analyzers/AnalyzersResources.resx:2
src/PublicApiAnalyzers/Core/Analyzers/PublicApiAnalyzerResources.resx:18
src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx:13
src/Text.Analyzers/Core/TextAnalyzersResources.resx:18
tools/FileIssues/Resources.resx:13
```

Reply:
@maryamariyan They can be used only for messages, not descriptions.